.devcontainer/setup.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash 
+set -euo pipefail
+
```

Comment:
Please add a comment of the exact purpose of setup.sh. Clarify things like if this is running on the host machine or in the container, and what its ultimate goal should be. My assumption is this runs on the host machine and should set up things in the container. 

Please clarify.